zebrad/tests/acceptance.rs

```diff
@@ -216,13 +226,17 @@ fn start_no_args() -> Result<()> {
     // Make sure the command was killed
     assert!(output.was_killed());
 
+    // Check that we have persistent sled database
+    let cache_dir = tempdir.join("state");
+    assert!(cache_dir.read_dir()?.count() > 0);
```

Comment:
Can we split this check out into a separate test, which just checks the `cache_dir` option?

See my earlier comment for some background:
https://github.com/ZcashFoundation/zebra/pull/939/files#r475980074